storage/aptosdb/src/state_merkle_db.rs

```diff
@@ -280,7 +294,8 @@ impl StateMerkleDb {
             );
         }
 
-        let batch = SchemaBatch::new();
+        let top_levels_batch = SchemaBatch::new();
```

Comment:
nit: might add some comment on why this is in its own batch

Reply:
Done